userspace/libsinsp/fdinfo.h

```diff
@@ -379,11 +379,21 @@ VISIBILITY_PRIVATE
 
 	void add_filename(const char* fullpath);
 
-	inline bool is_transaction()
+public:
+	inline bool is_transaction() const
 	{
 		return (m_usrstate != NULL); 
 	}
 
+	T* get_usrstate()
+	{
+		return m_usrstate;
+	}
+
+
+	
+
+private:
```

Comment:
`VISIBILITY_PRIVATE`?

Reply:
not if i can avoid it.

Trying to slowly move away from it to properly designed APIs, and if necessary at worst, helper friend classes.

Given that nobody here dependends on it (given that it compiles) I see no reason to add it.